setup.py

```diff
@@ -117,23 +134,18 @@
         'collidoscope>=0.5.2',  # Shaping (& Universal) profiles
                                 # 0.5.1 did not yet support python 3.11
                                 # (see https://github.com/googlefonts/fontbakery/issues/3970)
-        'defcon',  # Universal, UFO sources & OpenType profiles
         'stringbrewer',  # Shaping (& Universal) profiles
-        'ufolint',  # ufo_sources profile
-        'ufo2ft>=2.25.2',  # Shaping (& Universal), Opentype, ufo_sources profiles
+        f'ufo2ft{UFO2FT_VERSION}',  # Shaping
                            # 2.25.2 updated the script lists for Unicode 14.0
         'vharfbuzz>=0.2.0',  # Googlefonts, Shaping (& Universal) profiles
                              # v0.2.0 had an API update
     ],
     extras_require={
-        'docs': [
-            'recommonmark',
-            'sphinx >= 1.4',
-            'sphinx_rtd_theme',
-        ],
         'all': all_extras,
+        'docs': docs_extras,
         'googlefonts': googlefonts_extras,
         'fontval': fontval_extras,
+        'ufo-sources': ufo_sources_extras,
```

Comment:
@felipesanches the extra is already here

Reply:
`ufo-sources` matches the name of the profile that is shown in the CLI. I think we should keep it consistent.

Reply:
<img width="1265" alt="fbake" src="https://github.com/googlefonts/fontbakery/assets/2119742/c42a3e2b-02a1-4d9a-b5a4-2493614c3a1c">

Reply:
As already described in detail earlier in this thread, I consider this to be insufficient.

It is half past 10PM here in London, so I won't be able to work on it anymore today. I'll now merge this PR as-is (which is in a good state, except for the profile re-categorization of those UFO checks) so that you're free to move on with your work day in California.

Please do not cut a new release before I address this with some additional commits during my work day tomorrow.

Reply:
I prefer you hadn't merged. Merging with the intention of making your own changes doesn't seem like an good way of accepting contributions. I've rolled back all my recent PRs.